common/util.go

```diff
@@ -1009,3 +1009,11 @@ func ConvertGetTaskFailedCauseToErr(failedCause types.GetTaskFailedCause) error
 		return &types.InternalServiceError{Message: "uncategorized error"}
 	}
 }
+
+// GetTaskPriority returns priority given a task's priority class and subclass
```

Comment:
tbh I'm inclined to keep this next to the definitions of the priorities, since it's directly related to them.

semi-related though: is it difficult to make these different types?  relying on the iota'd consts as ints feels kinda wrong.
(if it's not trivial or you don't feel like it: nope!  this is how it was, leave it.)